extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -199,3 +204,96 @@ export const BackupPhraseWarningMessage = () => {
     </div>
   );
 };
+
+export const ScamAssetWarning = ({
+  domain,
+  code,
+  issuer,
+  image,
+}: {
+  domain: string;
+  code: string;
+  issuer: string;
+  image: string;
+}) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const warningRef = useRef<HTMLDivElement>(null);
+  const { isValidatingSafeAssetsEnabled } = useSelector(settingsSelector);
+
+  const closeOverlay = () => {
+    if (warningRef.current) {
+      warningRef.current.style.bottom = `-${POPUP_HEIGHT}px`;
+    }
+    setTimeout(() => {
+      dispatch(closeBlockedDomainWarning());
+    }, 300);
+  };
+
+  // animate entry
+  useEffect(() => {
+    if (warningRef.current) {
+      warningRef.current.style.bottom = "0";
+    }
+  }, [warningRef]);
```

Comment:
Is it possible to use `components/SlideupModal` here? It'll take care of this animation stuff for you